docs/Report.md

```diff
@@ -111,34 +88,31 @@ Then, confirmation of success is sent back until the razorpage redirects to itse
 <!-- Describe briefly the illustration, i.e., how you application is built, tested, released, and deployed. -->
 ### GitHub workflows
 
-To ensure the flow of the project, we use a tool developed by GitHub known as. GitHub Action, otherwise known as workflow. This will also entail when the workflows are activated and used.
+To ensure the flow of the project, we use a tool developed by GitHub known as. GitHub Action, otherwise known as a workflow. This will also entail when the workflows are activated and used.
 
 #### Build and Test
 
-The build and test workflow can be found in . The activity diagram shows how GitHub ensures what is merged into main. This workflow is run on a pull request every time a commit is made to the branch in the pull request. This is to ensure that main will still work by building the project with dotnet and tests made for the project.
- Because it runs the tests as well, it ensures that any incoming changes do not affect the functionality. If anything fails, it will stop and prevent the branch from merging into main.
-<figure>
-   <center> <img src="Images/BuildAndTest.png"
-         alt="BuildAndTest Activity diagram">
-    <figcaption>Fig.XX Build and test activity diagram</figcaption></center>
-</figure>
+The build and test workflow can be found [here](#BuildTest). The activity diagram shows how GitHub ensures what is merged into main does not damage it. 
+This workflow is run on a pull request every time a commit is made to the branch belonging to the pull request. 
+This ensures that main will stay functional by building and testing the project with dotnet and our test suite. If anything fails, it will stop and prevent the branch from merging into main.
+
+[BuildTest]: Images/BuildAndTest.png "Build and test activity diagram" 
+![Build and test activity diagram][image]{width=60%}
+
+
 
 #### Publish and release
 
-This workflow is made to automate the creation of a GitHub release when a tag is added (Appendix?). It will create a release of the tag. But first, the workflow builds a version for Windows, MacOS and Linux. After that, it will zip the files and add them to the release if a release was made.
-<figure>
-   <center> <img src="Images/PublishNewRelease.png"
-         alt="PublishNewRelease Activity diagram">
-    <figcaption>Fig.XX Publish new release activity diagram</figcaption></center>
-</figure>
+This workflow is made to automate the creation of a GitHub release when a tag is added (Appendix?). It will create a release of the tag. But first, the workflow in succession builds a version for Windows, MacOS and Linux. After that, it will zip the files and add them to the release if a release was made.
```

Comment:
Emphasising that we build in succession.

Reply:
Looks good